bookshelf/resources/views/list.php

```diff
@@ -12,48 +13,42 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-#}
-
-{% extends "base.html.twig" %}
-
-{% block content %}
+ob_start()
+?>
 
 <h3>Books</h3>
 <a href="/books/add" class="btn btn-success btn-sm">
   <i class="glyphicon glyphicon-plus"></i>
   Add book
 </a>
 
-{# [START book_list] #}
-{% for book in books %}
+<?php // [START book_list] ?>
+<?php foreach ($books as $i => $book): ?>
```

Comment:
It would be helpful to have a comment block at the top of each template with the expected variables described.

E.g., a list including book, it's type, short description. Like a method param doc.

Reply:
Great idea! Added.